.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,42 +14,43 @@
 
 ## Proposed change
 <!-- 
-    Describe the big picture of your changes here to communicate to the
-    maintainers why we should accept this pull request. If it fixes a bug
-    or resolves a feature request, be sure to link to that issue in the 
-    additional information section.
+  Describe the big picture of your changes here to communicate to the
+  maintainers why we should accept this pull request. If it fixes a bug
+  or resolves a feature request, be sure to link to that issue in the
+  additional information section.
 -->
 
 
 ## Type of change
 <!--
-    What types of changes does your PR introduce to our documention/website?
-    Put an `x` in the boxes that apply. You can also fill these out after
-    creating the PR.
+  What types of changes does your PR introduce to our documention/website?
```

Comment:
This isn't only for the documentation or website, right?

Reply:
There should only be one x here. If there are more than one ticks it's a sign that the PR should be split up.

Reply:
Oh yeah, I like that idea, thanks!

Reply:
Fix the docs stuff 🤦‍♂ 

Adjusted instruction to indicate one should split into multiple PRs to improve the speed and ease of the reviews.